docs/faq.md

```diff
@@ -427,3 +427,35 @@ You need to change kGREYConfigKeyScreenshotDirLocation in GREYConfiguration to c
 [[GREYConfiguration sharedInstance] setValue:@"screenshot_dir_path"
                                 forConfigKey:kGREYConfigKeyScreenshotDirLocation];
 ```
+
+**How do I run tests against a binary app?**
+
+Xcode 8 adds two new commands for building and running tests:
+
+- `build-for-testing` - Generates a xctestrun file for use with `test-without-building`.
+- `test-without-building` - Runs the tests from xctestrun against a precompiled app.
+
+For more information see `man xcodebuild.xctestrun`. The following commands work on the EarlGreyExample project.
+
+```
+$ cd Demo/EarlGreyExample
+$ pod install
+```
+
+```
```

Comment:
-sdk iphonesimulator9.3 as in https://github.com/google/EarlGrey/blob/master/Scripts/travis.sh#L44? Or do you feel that might be overdoing it?